Specs/DateTime/Spanish/DateTimePeriodExtractor.json

```diff
@@ -301,17 +246,6 @@
       }
     ]
   },
-  {
-    "Input": "Regresaré el próximo lunes a la tarde",
```

Comment:
We could change this test to `Regresaré el próximo lunes por la tarde` instead of deleting it.